examples/ftp-client-read/ftp_client_read.md

```diff
@@ -1,6 +1,6 @@
 # FTP client - Read file
 
-The FTP client is used to perform CRUD operation on remote files/directories using the FTP protocol. This sample includes getting file content with default configurations using the default port number.
+The FTP client can be used to read files on remote servers using the FTP protocol. An `ftp:Client` can be created with default configurations and the `get` API can be used to read the files. This returns the file content as a `byte[]` stream which can then be written to another file/output to console etc.
 
```

Comment:
**What does it do?**
The FTP client connects to a given FTP server, and then sends and receives files as byte streams.

**How to use it?** 
A FTP client is created by giving the host-name and required credentials. Once connected, `get` method is used to read files as byte streams from the FTP server.

**When to use it?**
Use this transfer files from a remote file system to a local file system.

Reply:
Updated in https://github.com/ballerina-platform/ballerina-distribution/pull/3749/commits/ed9908a4875590a75415d06bda29db0c03360fe9